src/components/VideoItem/VideoItem.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const VideoItemContainer = styled.div`
+    display: flex !important;
+    align-items: center !important;
+    cursor: pointer;
+    text-align: center;
+
+`;
+
+const Thumbnail = styled.img`
+    max-width: 200px;
+    max-height: 250px ;
+`;
+
+function VideoItem({id, title, thumbnail, description}){
+    return(
+        <VideoItemContainer id={id} key={Math.random()}>
```

Comment:
Instead of generating a random number why not just using the identifier?

```suggestion
        <VideoItemContainer id={id} key={`video-item-${id}`}>
```